pkgs/development/python-modules/ua-parser/default.nix

```diff
@@ -9,7 +9,7 @@ buildPythonPackage rec {
     sha256 = "0csh307zfz666kkk5idrw3crj1x8q8vsqgwqil0r1n1hs4p7ica7";
   };
 
-  buildInputs = [ pyyaml ];
+  propagatedBuildInputs = [ pyyaml ];
```

Comment:
```suggestion
  nativeBuildInputs = [ pyyaml ];
```
https://github.com/ua-parser/uap-python/blob/master/setup.py#L209

Reply:
That was great timing 😂 

It can be used at runtime here https://github.com/ua-parser/uap-python/blob/master/ua_parser/user_agent_parser.py#L528-L530

but after taking an extra deep look at the code I don't think that is a very standard code path and we probably can remove it completely.

Reply:
It's used in `setup.py`, that why it should be in `nativeBuildInputs`.

Reply:
> That was great timing joy

Just wait a little longer next time

Reply:
Not really. It is only used when building the regexes which I tried earlier and some packaging part complained that build_regexes is not a real subcommand.

Reply:
I removed it completely in #175397 and added a pythonImportsCheck

Reply:
The only reason it works without PyYAML is that the sdist includes `_regexes.py`.

Reply:
Yes, I know but I don't really know how to generate it and the code for it seems to be broken.